src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerEndpoint.java

```diff
@@ -78,12 +82,21 @@ public DockerServerEndpoint(String uri, String credentialsId) {
      *
      * <p>
      * Null to indicate whatever Docker picks by default.
+     * @deprecated use #getDockerHost
      */
     public @Nullable String getUri() {
         return uri;
     }
 
     /**
+     * Gets the endpoint in URI, such as "unix:///var/run/docker.sock".
+     */
+    public @Nonnull String getDockerHost() {
```

Comment:
Agree with @jtnord that the naming is inappropriate. The existing naming is `DockerServerEndpoint.uri`. If you are defining a default value for this property, then the corresponding method would be something like `DockerServerEndpoint.getEffectiveUri`. Compare the existing `DockerRegistryEndpoint.getEffectiveUrl`.